collections.yml

```diff
@@ -100,15 +100,6 @@
   name: Frontend
   name_cn: 前端技术
   items:
-    - ui-library # UI 库
-    - jquery-and-plugins # jQuery 及其插件
-    - bootstrap-theme # Bootstrap 主题
-    - mini-program-ui-library # 小程序 UI 库
-    - responsive-ui-frameworks # 响应式 UI 框架
-    - tailwindcss # Tailwind CSS 框架
-    - javascript-build-tool # 前端构建工具
-    - static-site-generator # 静态站点生成器
-    - electron-packaging-tools # Electron 打包工具
```

Comment:
@normal-coder , does it make sense to merge all these items into one item?

Reply:
I can't understand why. These technologies are already vertically oriented to specific business scenarios. Rubbing together doesn't create any instructive contrast.

BTW, the Pull Requests sent by @hncslwx and his friends don't explain why every time. Pull Request is always "No description provided.", I understand this is not acceptable :)

Reply:
@normal-coder @eyehwan Sorry to make confusion. We will fix this. before we push forward, we can take some time to review the whole picture of software classify and to create a clear manner of cooperation.